tests/Functional/TransactionTest.php

```diff
@@ -65,6 +62,24 @@ private function expectConnectionLoss(callable $scenario): void
         }
     }
 
+    private function killCurrentSession(): void
+    {
+        $databasePlatform = $this->connection->getDatabasePlatform();
+
+        [$currentProcessQuery, $killProcessStatement] = match (true) {
+            $databasePlatform instanceof AbstractMySqlPlatform => ['SELECT CONNECTION_ID()', 'KILL %s'],
+            default => self::markTestSkipped('Unsupported test platform.'),
+        };
+
+        $privilegedConnection = TestUtil::getPrivilegedConnection();
```

Comment:
Using priviledged connection here, assuming that's a requirement for other platforms (maybe oracle or SQL server?)